### PR TITLE
API-8467 - Add additional hlr rswag examples

### PR DESCRIPTION
**Pull request by nathanbwright**
## Description of change
Adds `/higher_level_reviews/validate` rswag spec.

## Original issue(s)
https://vajira.max.gov/browse/API-8467

### CODE REVIEW

File: modules/appeals_api/spec/docs/v2/hlr_spec.rb [hunk 1]
```diff
@@ -353,5 +353,166 @@
...
+            }
+          }
+        end
+
+        it 'Violates JSON schema' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '422', 'Error' do
```
Comment (cilestin) [inline modules/appeals_api/spec/docs/v2/hlr_spec.rb:487]: Not necessary now, but I'd like to see these more common errors that are the same across our endpoints made into support helpers so we could do things like `it_behaves_like 'Violates JSON Schema'`, etc :)

**Review by cilestin — APPROVED** [review]
Lookin' good!

Rswag's tests are a little hard to parse out, but isn't too bad overall. Just something to get used to.

**Comment by nathanbwright** [comment]
> Lookin' good!
> 
> Rswag's tests are a little hard to parse out, but isn't too bad overall. Just something to get used to.

Yeah, I had a little trouble reading them at first too... It would be nice to look into making them more reader friendly.
